### PR TITLE
Fixed bug with core collection key serialization [ECR-2710]

### PR DESCRIPTION
**Pull request by bullet-tooth**
## Overview

---
It was a bug with key serialization. We used little-endian byte order, but core uses big-endian one.

See: https://jira.bf.local/browse/ECR-2710

### Definition of Done

- [ ] There is no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ protected Schema createDataSchema(View view) {
     List<HashCode> stateHashes = super.getStateHashes(snapshot);
     // Log the state hashes, so that the values passed to the native part of the framework
     // are known.
-    logger.error("state hashes: {}", stateHashes);
+    logger.debug("state hashes: {}", stateHashes);
```
Comment (vitvakatu) [inline exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java]: Btw, it was one of the requirements of QA service (you can easily find it in wiki), not sure the reason though

Comment (dmitry-timofeev) [reply exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java]: Thanks, that's exactly the reason. If there isn't a comment clarifying that, please add.

Comment (bullet-tooth) [reply exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java]: Reverted this change. What is the reason for the `error` level requirement? Seeing errors in the log confuses me

**Comment by bullet-tooth** [comment]
Probably yes, it can be tested using system tests only. Do you have any clue how can we cover such things by unit or integration tests?

**Comment by dmitry-timofeev** [comment]
Probably, by writing to this (*internal core*) collection, but I'd abstain from that because that capability is likely to be temporary.

**Pull request merged**